modules/stanford_profile_styles/lib/scss/component/paragraph/react_paragraphs/_multirow.scss

```diff
@@ -36,8 +36,10 @@
 .react-paragraphs-row {
 
   // Single item per row on screens less than and equal to md.
+  // Added the clear for when media items are floated (right or left) within the wysiwyg.
   &.flex-container {
     @include grid-media-max('md') {
+      clear: both;
```

Comment:
Shouldn't it just clear on all `.flex-container` elements on every breakpoint? why isn't this provided by decanter?

Reply:
I did more testing and put it on all the breakpoints. I thought because it was specific to the floats in the paragraph. I can bring it up in the next decanter meeting?